comid/integrityregisters_test.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package comid
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/veraison/swid"
+)
+
+func prepareRegister(t *testing.T, iType string) (*IntegrityRegisters, error) {
```

Comment:
it doesn't look like the contents of the `IntegrityRegisters` produced by this function are used anywhere.  Looking at the tests below I suspect if you just returned an empty thing, e.g.:
```golang
	&IntegrityRegisters{
		m: map[IRegisterIndex]Digests{},
	},
```
all tests would still pass :)

Reply:
Use the returned types now!